lambdas/src/apis/explore/controllers/explore.ts

```diff
@@ -0,0 +1,181 @@
+import { Request, Response } from "express";
+import { DAOCache } from "../../../service/dao/DAOCache";
+import { ServerMetadata } from "decentraland-katalyst-commons/ServerMetadata";
+import { EntityType, Fetcher, Entity } from "dcl-catalyst-commons";
+import { noReject } from "decentraland-katalyst-utils/util";
+import { TimeRefreshedDataHolder } from "../../../utils/TimeRefreshedDataHolder";
+import { SmartContentClient } from "../../../utils/SmartContentClient";
+import ms from "ms";
+
+type ParcelCoord = [number, number];
+
+type RealmInfo = {
+  serverName: string;
+  layer: string;
+  usersCount: number;
+  usersMax: number;
+  userParcels: ParcelCoord[];
+};
+
+export type HotSceneInfo = {
+  id: string;
+  name: string;
+  baseCoords: ParcelCoord;
+  usersTotalCount: number;
+  parcels: ParcelCoord[];
+  thumbnail?: string;
+  projectId?: string;
+  creator?: string;
+  description?: string;
+  realms: RealmInfo[];
+};
+
+type Layer = {
+  name: string;
+  usersCount: number;
+  maxUsers: number;
+  usersParcels: ParcelCoord[];
+};
+
+type ServerStatus = {
+  name: string;
+  url: string;
+  layers: Layer[];
+};
+
+let realmsStatusCache: TimeRefreshedDataHolder<RealmInfo[]>;
+
+export async function realmsStatus(daoCache: DAOCache, req: Request, res: Response) {
+  // Method: GET
+  // Path: /realms
+
+  if (!realmsStatusCache) {
+    realmsStatusCache = new TimeRefreshedDataHolder(() => fetchRealmsData(daoCache), ms("1m"));
+  }
+
+  const realmsStatusData = await realmsStatusCache.get();
+
+  res.status(200).send(realmsStatusData);
+}
+
+let hotSceneCache: TimeRefreshedDataHolder<HotSceneInfo[]>;
+
+export async function hotScenes(daoCache: DAOCache, contentClient: SmartContentClient, req: Request, res: Response) {
+  // Method: GET
+  // Path: /hot-scenes
+
+  if (!hotSceneCache) {
+    hotSceneCache = new TimeRefreshedDataHolder(() => fetchHotScenesData(daoCache, contentClient), ms("1m"));
+  }
+
+  const hotScenesData = await hotSceneCache.get();
+
+  res.status(200).send(hotScenesData);
+}
+
+async function fetchRealmsData(daoCache: DAOCache): Promise<RealmInfo[]> {
+  const statuses = await fetchCatalystStatuses(daoCache);
+
+  return statuses
+    .flatMap((server) =>
+      server.layers.map((layer) => ({
+        serverName: server.name,
+        url: server.url,
+        layer: layer.name,
+        usersCount: layer.usersCount,
+        usersMax: layer.maxUsers,
+        userParcels: layer.usersParcels,
+      }))
+    )
+    .sort((realm1, realm2) => realm2.usersCount - realm1.usersCount);
+}
+
+async function fetchHotScenesData(daoCache: DAOCache, contentClient: SmartContentClient): Promise<HotSceneInfo[]> {
+  const statuses = await fetchCatalystStatuses(daoCache);
+  const tiles = getOccupiedTiles(statuses);
+
+  if (tiles.length > 0) {
+    const scenes = await contentClient.fetchEntitiesByPointers(EntityType.SCENE as any, tiles);
+
+    const hotScenes: HotSceneInfo[] = scenes.map(getHotSceneRecordFor);
```

Comment:
Would it make sense to first slice the scenes up to a given number? We could potentially have A LOT of scenes here, and it might make sense to add a maximum